show/main.py

```diff
@@ -3051,53 +3051,46 @@ def ztp(status, verbose):
     run_command(cmd, display_cmd=verbose)
 
 #
-# show features
-#
-@cli.command('features')
-def features():
-    """Show status of optional features"""
-    config_db = ConfigDBConnector()
-    config_db.connect()
-    header = ['Feature', 'Status']
-    body = []
-    status_data = config_db.get_table('FEATURE')
-    for key in status_data.keys():
-        body.append([key, status_data[key]['status']])
-    click.echo(tabulate(body, header))
-
-#
-# 'container' group (show container ...)
+# 'feature' group (show feature ...)
 #
-@cli.group(name='container', invoke_without_command=False)
-def container():
-    """Show container"""
+@cli.group(name='feature', invoke_without_command=False)
+def feature():
+    """Show feature status"""
     pass
 
 #
-# 'feature' group (show container feature ...)
+# 'status' subcommand (show feature status)
 #
-@container.group(name='feature', invoke_without_command=False)
-def feature():
-    """Show container feature"""
-    pass
+@feature.command('status', short_help="Show feature status")
+@click.argument('feature_name', required=False)
+def autorestart(feature_name):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    header = ['Feature', 'Status', 'AutoRestart']
```

Comment:
'Status' should change to 'State' to align with terminology from `config` command